client/web/src/hooks/node-data.ts

```diff
@@ -179,6 +188,19 @@ export default function useNodeData() {
         if (data?.AdvertisingExitNode !== d.AdvertiseExitNode) {
           incrementMetric(d.AdvertiseExitNode ? "web_client_advertise_exitnode_enable" : "web_client_advertise_exitnode_disable")
         }
+        // useExitNode is the ID of the exit node to use
+        if (data?.UsingExitNode?.ID !== d.UseExitNode) {
+          console.log(`${data?.UsingExitNode} ${d.UseExitNode}`)
+          incrementMetric(d.UseExitNode ? "web_client_use_exitnode_enable" : "web_client_use_exitnode_disable")
+        }
+        // going from no routes in old data -> advertised routes in new data
+        if ((data?.AdvertisedRoutes == null || data?.AdvertisedRoutes.length === 0) && (d.AdvertiseRoutes && d.AdvertiseRoutes.length > 0)) {
+          incrementMetric("web_client_advertise_routes_enable")
+        }
+        // going from advertised routes in old data -> no routes in new data 
+        if (data?.AdvertisedRoutes && data?.AdvertisedRoutes.length > 0 && (d.AdvertiseRoutes == null || d.AdvertiseRoutes?.length === 0)) {
+          incrementMetric("web_client_advertise_routes_disable")
+        }
```

Comment:
I wonder if "enable"/"disable" is really the right thing here - Thoughts on `web_client_advertised_routes_add` and `web_client_advertised_routes_remove`

Or maybe we really only care about `web_client_advertised_routes_changed`

Reply:
Maybe a question for sam

Reply:
Ah good call, yeah think I'll defer to Sam on this one. Wonder if we might even want both (e.g., when something has been added / removed / changed and when advertising has been first enabled or fully disabled).